en/docs/monitoring-and-insights/view-logs.md

```diff
@@ -0,0 +1,106 @@
+
+## Runtime logs
+
+Choreo runtime logs provide insights into project-level and component-level logs, covering application, gateway, system, and access logs. The runtime logs streamline the debugging process by centralizing diverse log sources.
+
+You can view runtime logs for the past 30 days by default when you access the runtime logs page. Choreo also allows precise filtering of runtime logs based on parameters such as log level (error, warn, info, debug), log type (application, gateway, system, access), and environment (development, staging, production). These granular filters effectively simplify the debugging process. 
+
+To access runtime logs, follow the steps below:
+
+1. Sign in to [Choreo](https://console.choreo.dev/) using your Google, GitHub, or Microsoft account.
+2. In the left navigation menu, click **Logs** and then click **Runtime Logs**. This displays runtime logs for the past 30 days by default.
+
+    ![Runtime logs](../assets/img/monitoring-and-insights/view-logs/runtime-logs.png)
+
+    You can apply the necessary filter criteria to view logs according to your requirements.
+
+## Audit logs
+
+Audit logs, also called audit trails, enhance security, ensure compliance, provide operational insights, and help manage risks. 
+
+In Choreo, an audit log records organization-level user-specific operations performed via the Choreo Console. It also captures the timestamp and the outcome of the action. 
+
+As of now, Choreo captures the following user-specific operations as audit logs:
+
+- Project creation, update, and deletion.
+- Component creation, update, and deletion.
+- Component promotion initiation.
+- Component version creation.
+- Component deployment, redeployment, and undeployment initiation for all components other than REST API Proxy components.
+- Component API access mode update.
+- Enabling and disabling component auto-deployment on commit. 
+- Component build configuration update.
+- Component endpoint creation, update, and deletion.
+- Organization user management.
+- On-premises key management.
+- Project-level configuration management.
+
+In Choreo, organization administrators are allowed to view audit logs by default. If other members need to access organization-specific audit logs, the administrator can create a role with the relevant permission and assign it to members. For step-by-step instructions on how to create and assign a role with relevant permission, see [Manage audit log access](#manage-audit-log-access).
+
+To view audit logs, follow these steps:
+
+1. Sign in to [Choreo](https://console.choreo.dev/) using your Google, GitHub, or Microsoft account.
+2. In the Choreo Console, go to the top navigation menu and click **Organization**.
+   
+    !!! tip
+         As of now, you can only view organization-level audit logs.
+
+3. In the left navigation menu, click **Logs** and then click **Audit Logs**. This displays audit logs for the past 30 days by default.
```

Comment:
Since we mention that it shows logs for the past 30 days by default, do we need to say that they can change this period using the date selectors?

Reply:
Had a sentence after the screenshot to indicate the possibility of applying filter criteria to view logs depending on specific requirements. Moved this before the screenshot and rephrased it accordingly.